telebot/types.py

```diff
@@ -822,7 +828,11 @@ def __init__(self, id, type, title=None, username=None, first_name=None,
         self.personal_chat: Optional[Chat] = personal_chat
         self.birthdate: Optional[Birthdate] = birthdate
         self.can_send_paid_media: Optional[bool] = can_send_paid_media
-        self.can_send_gift: Optional[bool] = can_send_gift
+        self.accepted_gift_types: AcceptedGiftTypes = accepted_gift_types
+        self.can_send_gift: Optional[bool] = None
```

Comment:
Propose to make propery with deprecation warning. Like in Message.voice_chat_scheduled, for example.